src/main/java/com/aws/greengrass/clientdevices/auth/iot/CertificateRegistry.java

```diff
@@ -101,13 +101,6 @@ public void updateCertificate(Certificate certificate) {
         runtimeConfiguration.putCertificate(certificateToCertificateV1DTO(certificate));
     }
 
-    /**
-     * Returns the PEMs that are stored for all the certificate ids that are being tracked.
-     */
-    public String[] getAllCertificateIds() {
```

Comment:
As much as I :heart: seeing red, I think we'll still want this for the removal flow.

Reply:
I am still removing it and basing it off the certificates attached to things. See https://github.com/aws-greengrass/aws-greengrass-client-device-auth/pull/166/files#diff-6161e346c7f85b92277837560e988c3304ad8a2f83eb92b5525a2e92e7fc3dd3R214